src/main/java/com/yanna/stepanova/mapper/CartItemMapper.java

```diff
@@ -0,0 +1,28 @@
+package com.yanna.stepanova.mapper;
+
+import com.yanna.stepanova.config.MapperConfig;
+import com.yanna.stepanova.dto.cartitem.CartItemDto;
+import com.yanna.stepanova.dto.cartitem.CartItemQuantityRequestDto;
+import com.yanna.stepanova.dto.cartitem.CreateCartItemRequestDto;
+import com.yanna.stepanova.model.CartItem;
+import org.mapstruct.AfterMapping;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
+
+@Mapper(config = MapperConfig.class, uses = BookMapper.class)
+public interface CartItemMapper {
+    @Mapping(target = "book", source = "bookId", qualifiedByName = "bookFromId")
+    CartItem toModel(CreateCartItemRequestDto requestDto);
+
+    CartItemDto toDto(CartItem cartItem);
```

Comment:
```suggestion
    @Mapping(source = "book.id", target = "bookId")
    @Mapping(source = "book.title", target = "bookTitle")
    CartItemDto toDto(CartItem cartItem);
```

instead of setBook()